MeshLib/MeshGenerators/MeshLayerMapper.cpp

```diff
@@ -60,6 +61,10 @@ MeshLib::Mesh* MeshLayerMapper::createStaticLayers(MeshLib::Mesh const& mesh, st
 	std::vector<MeshLib::Node*> new_nodes(nNodes + (nLayers * nNodes));
 	std::vector<MeshLib::Element*> new_elems;
 	new_elems.reserve(nElems * nLayers);
+	MeshLib::Properties properties;
+	boost::optional<PropertyVector<int> &> materials = 
+		properties.createNewPropertyVector<int>("MaterialIDs", MeshLib::MeshItemType::Cell);
+	materials->reserve(nElems * nLayers);
```

Comment:
I just talked to Lars and we both dont get it .... I create a properties object and immediatly create the mat-vector. how can the vector be uninitalised? judging from the implementation of ```createPropertyVector``` this is only possible if there is already a vector of the same name, which cannot happen here because the property object has been created just for the purpose of creating the mat vector.